foundation/foundation-mda/src/main/resources/templates/deployment/hive-metastore-service/hive.properties.vm

```diff
@@ -0,0 +1,4 @@
+
+# Password and other sensitive information should be encrypted using krausening
+# (See Here: https://github.com/TechnologyBrewery/krausening/tree/dev/krausening/#krausening-in-four-pints-leveraging-jasypt-for-encryptingdecrypting-properties)
+HIVE_USERNAME=hive
```

Comment:
S: Properties files should generally follow Java package/variable naming conventions, i.e. `group.aSubGroup.propertyName`. Here I would suggest `hive.db.username`

Reply:
Ok, I'm revisiting this again.  _Actually_, I think it's worth a little more thought toward what organization of properties makes sense in the long run.  This is configuring the Spark metastore. So it probably makes more sense to name the file `spark.properties` and the property `metastore.db.username`.  We just need to look at pre-existing config files that might be called `spark.properties` and make sure it all  works together.

Reply:
looking at the aissemble repo and downstream project repo that uses Spark doesn't have config file called spark.properties or anything similar in that sense. 
I agree being spark. but since this is more of infrastructure. I think it also makes sense to name it as `spark-infrastructure.properties` unless anyone objects

Reply:
Would update the template locations within foundation mda to reflect the new file name, like move the `hive.properties.vm` from `hive-metastore-service` to `spark-infrastructure`